### PR TITLE
Adding LICENSE file to project

### PR DESCRIPTION
**Pull request by kshithijiyer**
fixes: #83

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -0,0 +1,201 @@
...
+      To apply the Apache License to your work, attach the following
+      boilerplate notice, with the fields enclosed by brackets "[]"
+      replaced with your own identifying information. (Don't include
+      the brackets!)  The text should be enclosed in the appropriate
+      comment syntax for the file format. We also recommend that a
+      file or class name and description of purpose be included on the
+      same "printed page" as the copyright notice for easier
+      identification within third-party archives.
+
+   Copyright [yyyy] [name of copyright owner]
```
Comment (adambkaplan) [inline LICENSE]: Remove APPENDIX above, update copyright as follows:

```suggestion
   Copyright 2019 Red Hat, Inc.
```

Comment (kshithijiyer) [reply LICENSE]: Done

**Comment by adambkaplan** [comment]
squash commits, otherwise looks good

**Comment by kshithijiyer** [comment]
Sure! You can squash and merge it.

**Comment by adambkaplan** [comment]
@kshithijiyer I should have clarified - please squash your commits. With a few exceptions, we require that contributors squash their commits before we merge each PR.

**Comment by kshithijiyer** [comment]
> @kshithijiyer I should have clarified - please squash your commits. With a few exceptions, we require that contributors squash their commits before we merge each PR.
Done.

**Comment by adambkaplan** [comment]
/hold

Issues with Imagestreams and builds - needs investigation of the OpenShift controller manager

**Comment by adambkaplan** [comment]
Builds tests are referencing an obsolete Imagestream. Need to fix that
first.

**Comment by kshithijiyer** [comment]
> /hold
> 
> Issues with Imagestreams and builds - needs investigation of the OpenShift controller manager

Don't we need to unhold it?

**Pull request merged**